ci-constraints-requirements.txt

```diff
@@ -113,7 +117,7 @@ markupsafe==3.0.2 ; python_full_version >= '3.9'
     # via jinja2
 mypy==1.4.1 ; python_full_version < '3.8'
     # via cryptography (pyproject.toml)
-mypy==1.15.0 ; python_full_version == '3.8.*'
+mypy==1.14.1 ; python_full_version == '3.8.*'
```

Comment:
Why did this go down?

Reply:
Because the old one was wrong, but we never noticed because we don't test mypy on an old Python.

If you look below, you'll see 1.15 requires >=3.9